packages/gitbook-docs/src/markdoc/schema/link.ts

```diff
@@ -18,6 +19,15 @@ export const link: Schema = {
   },
   transform: (node, config) => {
     const attrs = node.transformAttributes(config);
+    /* eslint-disable functional/no-expression-statements */
+    /* eslint-disable functional/immutable-data */
+    if (attrs.href && attrs.href.includes('mailto:')) {
+      // delete everything before the 'mailto:'
+      attrs.href = attrs.href.replace(/.*mailto:/, 'mailto:');
+    }
+    /* eslint-enable functional/no-expression-statements */
+    /* eslint-enable functional/immutable-data */
+
```

Comment:
Instead change the behaviour of `LinkAttr` ([/packages/gitbook-docs/src/markdoc/attributes.ts#L36-L55](https://github.com/pagopa/developer-portal/blob/23e4942/packages/gitbook-docs/src/markdoc/attributes.ts#L36-L55))

```suggestion
```